web/vtadmin/src/components/routes/Workflows.tsx

```diff
@@ -112,13 +112,53 @@ export const Workflows = () => {
                             {/* TODO(doeg): add a protobuf enum for this (https://github.com/vitessio/vitess/projects/12#card-60190340) */}
                             {['Error', 'Copying', 'Running', 'Stopped'].map((streamState) => {
                                 if (streamState in row.streams) {
+                                    var numThrottled = 0;
+                                    var throttledApp = '';
+                                    var throttledFrom: vttime.ITime | null | undefined;
                                     const streamCount = row.streams[streamState].length;
+                                    var streamDescription: string;
+                                    switch (streamState) {
+                                        case 'Error':
+                                            streamDescription = 'failed';
```

Comment:
I don't like this terminology since errors are relatively common and we almost immediately retry. `Error` is better than `failed` IMO as failed is typically a persistent/permanent state as it's a post state (failed vs failure).

Reply:
This nomenclature has been in vtadmin for a while, so not changing it in this PR, in case there was a reason for it.